make-typed-request.js

```diff
@@ -36,7 +41,8 @@ function makeTypedRequest(treq, opts, cb) {
             httpVersion: resp.httpVersion,
             headers: resp.headers,
             statusCode: resp.statusCode,
-            body: resp.body
+            body: resp.body,
+            requestUrl: reqOpts.url
```

Comment:
Trying to compare the results in the unit test. Not sure how to get the modified request in the test.